SupportClient/SupportClient/Controllers/FeedbackViewController.swift

```diff
@@ -0,0 +1,219 @@
+//
+
+import UIKit
+import SnapKit
+import DZNEmptyDataSet
+import BSImagePicker
+import Photos
+
+class FeedbackViewController: UIViewController {
+
+    private var attachedErrorFiles = [PHAsset]()
+    private var collectionView: UICollectionView!
+    
+    private let messageLabel = UILabel()
+    private let messageTextField = UITextField()
+    private let typeLabel = UILabel()
+            
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.backgroundColor
+        setupNavigationBar()
+        setupCollectionView()
+        setupConstraints()
+    }
+    
+    func setupNavigationBar() {
+        self.navigationController?.navigationBar.barTintColor = UIColor.navigationTintColor
+        let attributes = [
+            NSAttributedString.Key.font: UIFont._21RobotoMedium!,
+            NSAttributedString.Key.foregroundColor: UIColor.titleColor
+        ]
+        let attributedTitle = NSAttributedString(string: "New Feedback", attributes: attributes)
+        title = attributedTitle.string
+        navigationItem.leftBarButtonItem = UIBarButtonItem(
+            title: "Cancel",
+            style: UIBarButtonItem.Style.plain,
+            target: self,
+            action: #selector(handleNavigationBarLeftTap)
+        )
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            title: "Send",
+            style: UIBarButtonItem.Style.plain,
+            target: self,
+            action: #selector(handleNavigationBarRightTap)
+        )
+    }
+    
+    func setupCollectionView() {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        collectionView.emptyDataSetSource = self
+        collectionView.emptyDataSetDelegate = self
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.backgroundColor = UIColor.backgroundColor
+        collectionView.showsHorizontalScrollIndicator = false
+        collectionView.register(ErrorFileCollectionViewCell.self, forCellWithReuseIdentifier: ErrorFileCollectionViewCell.reuseID)
+        view.addSubview(collectionView)
+    }
+    
+    func setupConstraints() {
+        NSLayoutConstraint.activate([
+            collectionView.heightAnchor.constraint(equalToConstant: UIScreen.main.bounds.height / ErrorFileCollectionViewCell.imageResizingRatio),
+            collectionView.widthAnchor.constraint(equalTo: view.widthAnchor),
+            collectionView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 50),
+        ])
+    }
+    
+    // TODO: animation does not work after presenting ImagePicker sometimes
+    @objc func handleNavigationBarLeftTap() {
+        self.dismiss(animated: true, completion: nil)
```

Comment:
Can u put a `// TODO:` wherever ur leaving stuff to be done in future PRs? I always get anxious that I'm like possibly forgetting something. Ik this is an essential action so we'll most prob not forget, but i think its good to get in the habit.